### ISSUE TITLE
Retiring redundant `ddev-contrib` 

### ISSUE THREAD

**Issue by tyler36** [drud/ddev-contrib#204]
With the upcoming release of `1.19`, Postgre support is in-build making the recipe here redundant.
There is also a move towards `ddev get` and "official" versions.

We should consider how we want to "retire" redundant contribs while still maintaining searchability for users that may require older versions of DDEV.

- Do we flag items with DDEV version tags?
- Provide links to the new service and/or relevant documentaion?
- Add migration guides?

Just thinking out loud.

**Comment by rfay** [comment]
Thanks for paying attention! 

* I recommend a "RETIRED, look over here" statement at the top. Then after months or a year remove.
* I hope that all the most important services will become independent `ddev get` recipes within just a few months.

**Comment by tyler36** [comment]
Yes, I'm excited about the move to `ddev get`. Didn't know if you had a process lined up yet since I was playing with the new Postgres earlier.

**Comment by tyler36** [comment]
Closing.

README contains the following warning/hint: 

> MOST OF THESE recipes have been made obsolete by DDEV add-ons. Please look for add-ons before using anything here. ddev get --list --all
> 
> See [docs](https://ddev.readthedocs.io/en/stable/users/extend/additional-services/) and ddev get --list for official add-ons, ddev get --list --all for all add-ons.
